### ISSUE TITLE
Test for WCAG failure F77: Duplicate values of type ID

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#164]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/F77
This describes a failure condition where duplicate ID errors are known to cause problems for assistive technologies when they are trying to interact with content. Duplicate values of type ID can be problematic for user agents that rely on this attribute to accurately convey relationships between different parts of content to users. For example, a screen reader may use ID values to identify the applicable header content for a data cell within a data table, or an input control to which a given label applies. If these values are not unique, the screen reader will be unable to programmatically determine which headers are associated with the data cell or which control is associated with which label or name.

Checking that ID attribute values are unique within a document can be done by validating the document against its schema, because the schema defines which attributes contain document-wide unique identifiers.

Note 1: In most markup languages, ID values are attribute values, for example in HTML and XHTML
## Test procedure
1. Check that all values of type ID are unique in the Web page
